### ISSUE TITLE
How to get index no of row after sorted the table?

### ISSUE THREAD

**Issue by Praveene7** [olifolkerd/tabulator#649]
After sorted could not get index number of the row. I can only get id of the row but need position of the row.

**Comment by olifolkerd** [comment]
at the moment this is not possible. i will look to add a ***getPosition*** function to the row component in the next release.

What is you usage case for this? it will help me tailor the funcction to your needs.

Cheers

Oli

**Comment by Praveene7** [comment]
Hi Oli,

If I click on a cell have to check some other cell on same row. If that cell is empty show message like the "cell is empty on row position". this should be happen after sorting.

**Comment by olifolkerd** [comment]
thanks for the usage case, that will help me to tailor the function to your needs

**Comment by olifolkerd** [comment]
Hey @Praveene7 

I have just added the ***getPosition*** function to the 3.4 branch.

You can now find the position of a row in the table by calling the ***getPosition*** function on its row component:
```js
var rowPosition = row.getPosition(true); //return the position of the row in the filtered/sorted data
```

This will be available in the 3.4 release in a couple of weeks time.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Tabulator version 3.4 has now been officially released!

You can find details of the new release in the [News Section](http://tabulator.info/news)

Full documentation of all the new features can be found [Here](http://tabulator.info/docs/3.4)

Cheers

Oli :)
